app/src/main/java/io/github/omisie11/spacexfollower/ui/capsules/CapsuleDetailFragment.kt

```diff
@@ -28,7 +27,6 @@ class CapsuleDetailFragment : Fragment() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        sharedElementEnterTransition = MaterialContainerTransform(requireContext())
```

Comment:
This deletes previous animations when navigating from list of Capsules to current capsule details screen.
Why did You deleted that?
Same question for other Details screens

Reply:
I was experimenting with the container to the details page and removed the code so by mistake.sorry mate

Reply:
No problem, so bring back those animations in Capsules, Cores and Launches Details 😃